docker/powergate-image.yaml

```diff
@@ -0,0 +1,5 @@
+version: "3.7"
+
+services:
+  powergate:
+    image: textile/powergate:latest
```

Comment:
At release time, this is updated to the release tag version and included in the released artifacts. Makefile is update to pull this in instead of `powergate-build-context.yaml`